planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -75,6 +75,9 @@ BehaviorPathPlannerNode::BehaviorPathPlannerNode(const rclcpp::NodeOptions & nod
       create_publisher<MarkerArray>("~/drivable_area_boundary", 1);
   }
 
+  left_bound_publisher_ = create_publisher<Marker>("~/left_bound", 1);
+  right_bound_publisher_ = create_publisher<Marker>("~/right_bound", 1);
```

Comment:
```suggestion
  left_bound_publisher_ = create_publisher<Marker>("~/debug/left_bound", 1);
  right_bound_publisher_ = create_publisher<Marker>("~/debug/right_bound", 1);
```
We are currently working on documentation for the namespace and are moving forward with debugging messages with `~/debug/`.

Reply:
done